From 5fa3ca52844b7ae6d70e71d6f7892e85e44a394f Mon Sep 17 00:00:00 2001
From: Sylvain Munaut <tnt@246tNt.com>
Date: Sun, 13 May 2007 16:12:23 +0200
Subject: [PATCH 21/21] powerpc: Fix errors in bcom -> bcom_eng renaming.

At some point I reversed too many changes and forgot to
reapply the renaming to the debug functions ...

Signed-off-by: Sylvain Munaut <tnt@246tNt.com>
---
 arch/powerpc/sysdev/bestcomm/bestcomm.c |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/powerpc/sysdev/bestcomm/bestcomm.c b/arch/powerpc/sysdev/bestcomm/bestcomm.c
index 68c8e39..10fb476 100644
--- a/arch/powerpc/sysdev/bestcomm/bestcomm.c
+++ b/arch/powerpc/sysdev/bestcomm/bestcomm.c
@@ -47,10 +47,10 @@ void
 bcom_dump_status(void)
 {
 	int i;
-	struct mpc52xx_sdma __iomem *r = bcom->regs;
+	struct mpc52xx_sdma __iomem *r = bcom_eng->regs;
 
 	BCOM_DPRINTK("BestComm status dump (pa=%08lx, va=%p)\n",
-			bcom->regs_base, bcom->regs);
+			bcom_eng->regs_base, bcom_eng->regs);
 	BCOM_DPRINTK(" taskBar         = %08x\n", in_be32(&r->taskBar));
 	BCOM_DPRINTK(" currentPointer  = %08x\n", in_be32(&r->currentPointer));
 	BCOM_DPRINTK(" endPointer      = %08x\n", in_be32(&r->endPointer));
@@ -97,11 +97,11 @@ bcom_dump_task(int task)
 {
 	int i;
 	u32 *p;
-	struct bcom_tdt *tdt = &bcom->tdt[task];
+	struct bcom_tdt *tdt = &bcom_eng->tdt[task];
 
 	BCOM_DPRINTK("Task dump %d\n", task);
-	BCOM_DPRINTK(" tcr          = %04hx\n", bcom->regs->tcr[task]);
-	BCOM_DPRINTK(" tdt          = %p\n", &bcom->tdt[task]);
+	BCOM_DPRINTK(" tcr          = %04hx\n", bcom_eng->regs->tcr[task]);
+	BCOM_DPRINTK(" tdt          = %p\n", &bcom_eng->tdt[task]);
 	BCOM_DPRINTK(" tdt->start   = %08x\n", tdt->start);
 	BCOM_DPRINTK(" tdt->stop    = %08x\n", tdt->stop);
 	BCOM_DPRINTK(" tdt->var     = %08x\n", tdt->var);
-- 
1.5.1.2

